avocado/core/sysinfo.py

```diff
@@ -213,6 +214,45 @@ def stop(self):
         return retcode
 
 
+class JournalctlWatcher(Collectible):
+
+    """
+    Track the content of systemd journal into a compressed file.
+
+    :param logf: Basename of the file where output is logged (optional).
```

Comment:
Oh, now I notice that `LogFile` does the same (documents `__init__` parameters on the class docstring).  Anyway, I really think it's not the best place for it.